Documentation/network.md

```diff
@@ -123,13 +123,13 @@ homeassistant login:
 
 From there you use the `nmcli` configuration tool.
 
-- `# nmcli con show` will list the "HassOS default" connection in use.
-- `# nmcli con show "HassOS default"` will list all the properties of the connection.
+- `# nmcli con show` will list the "my-network" connection in use.
+- `# nmcli con show "my-network"` will list all the properties of the connection.
```

Comment:
Not sure why this change is needed? On my end the default network name in a newly installed system is `HassOS default`:

```
# nmcli con show
NAME            UUID                                  TYPE      DEVICE 
HassOS default  f62bf7c2-e565-49ff-bbfc-a4cf791e6add  ethernet  eth0   
```

Reply:
I agree. Could be that very old installation are not like this. Anyway today we have an UI for this, maybe we don't need this add all and should promote our CLI

Reply:
Ok, that probably would need a new merge request. Closing this one.